Sources/Transaction/Amount.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+import Foundation
+
+public struct Amount {
+    public let value: UInt64
+    public let tokenId: TokenId
+}
+
+extension Amount {
+    public init(value: UInt64, token: TokenId) {
+        self.value = value
+        self.tokenId = token
+    }
+
+    public init(_ value: UInt64, _ tokenId: TokenId) {
+        self.value = value
+        self.tokenId = tokenId
+    }
```

Comment:
why do we need both of these?

Reply:
the second one has a shorter function signature which im using in a few places to keep the lines more readable.

Reply:
Sure...  But if the second one is sufficient, wondering why we need the first?   It's not a huge deal to have both, but a minor nit would be to make the parameter names match for tokenId.

Reply:
What do you think of:
`public init(_ value: UInt64, in tokenId: TokenId)`  

so the client code will look like:

` Amount(value, in: .MOB)` 

or

`Amount(value, in: tokenId)`?

Reply:
ok will do this

Reply:
done